### DISCUSSION TITLE
Sharing preview of my book on LazyVim

### DISCUSSION THREAD

**Discussion by dusty-phillips** [LazyVim/LazyVim#3066] [General]
I've written a book on LazyVim and am in the final editing stages. I'm looking for feedback and review if anyone is interested.

_LazyVim for Ambitious Developers_ is intended to introduce modal editing to new-to-vim users coming from e.g. VSCode or Sublime Text. I can all but guarantee that it also contains at least one tidbit of information that will be new to even experienced Neovim developers. 

I believe that LazyVim is THE best developer experience available today, and I want to help promote it to the community that hasn't embraced Neovim yet.

The first three chapters are already available, and I will be loading more as I finish editing them down. You can access the book in web format at https://lazyvim-ambitious-devs.phillips.codes. If you think this is valuable work, please consider supporting me on Patreon or Github sponsors.

I haven't decided for sure whether to make a print/e-book edition, but the current plan is to self-publish it if interest is high enough. My other books have gone through traditional publishers, but my experience with them has not made me want to reach out to them again. ;-)

**Comment by rubiin** [comment]
Looks good mate. You should also consider making a pdf version for offline viewing for maximum reach :)

**Reply by dusty-phillips** [reply]
Thank you! I added a service worker, so the site should work in offline mode, but I'll also dig into PDF generation.

**Reply by dusty-phillips** [reply]
Managed a quick and dirty PDF. Link is in the left sidebar on the course page.

**Comment by jjansen72** [comment]
Great timing! I started my NeoVim journey 2 weeks ago with LazyVim. Just today I was wondering (and been a bit confused about Leap) what jump-plugin LazyVim uses per default (which I now know having read the free chapters - flash). So far I find the chapters very good. I hope the other chapters are mostly done, because I'm really looking forward to it.

Count me in! In terms of publishing, I think your website and pdf are fine. I'm good with everything that accepts paypal and most are probably as well.

**Comment by dusty-phillips** [comment]
Just wanted to resurface this thread, as I'm now up to eight chapters published (and another two on Patreon) on the book, so the content is hopefully becoming relevant to more people! https://lazyvim-ambitious-devs.phillips.codes/

**Reply by folke** [reply]
Looks really good! Feel free to open a PR to add a link to it.
